sharktank/sharktank/layers/kv_cache.py

```diff
@@ -64,7 +63,6 @@ def __init__(
         self.transformer_block_count = transformer_block_count
         self.attn_head_count = attn_head_count
         self.attn_head_dim = attn_head_dim
-        self.cache_partition_count = cache_partition_count
```

Comment:
Lets go ahead and keep the member variable but remove the argument instead setting this as 2 to avoid using unnamed `2`'s below. You can add a comment that it should remain fixed if you feel that its ambiguous.

Reply:
done